numpy/core/tests/test_item_selection.py

```diff
@@ -18,10 +18,9 @@ def test_simple(self):
         index_arrays = [np.empty(0, dtype=np.intp),
                         np.empty(tuple(), dtype=np.intp),
                         np.empty((1, 1), dtype=np.intp)]
-        real_indices = {}
-        real_indices['raise'] = {-1:1, 4:IndexError}
-        real_indices['wrap'] = {-1:1, 4:0}
-        real_indices['clip'] = {-1:0, 4:1}
+        real_indices = {'raise': {-1: 1, 4: IndexError},
```

Comment:
Oh wow, that is confusing. I had to read like 5 times to see it, but
they're not slices, they're dict key/values.
On Nov 7, 2015 8:37 AM, "Jaime" <notifications@github.com> wrote:

> In numpy/core/tests/test_item_selection.py
> <https://github.com/numpy/numpy/pull/6649#discussion_r44214162>:
>
> > @@ -18,10 +18,9 @@ def test_simple(self):
> >          index_arrays = [np.empty(0, dtype=np.intp),
> >                          np.empty(tuple(), dtype=np.intp),
> >                          np.empty((1, 1), dtype=np.intp)]
> > -        real_indices = {}
> > -        real_indices['raise'] = {-1:1, 4:IndexError}
> > -        real_indices['wrap'] = {-1:1, 4:0}
> > -        real_indices['clip'] = {-1:0, 4:1}
> > +        real_indices = {'raise': {-1: 1, 4: IndexError},
>
> I think the convention is no spaces in slices, I.e. after the colon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/6649/files#r44214162>.
>